src/text/fi.json

```
@@ -144,7 +144,7 @@
   "catalogue_emptyCatalogue": "Kalustoluettelossa ei ole ajoneuvoja.",
   "catalogue_addCatalogue": "Lisää kalustoluettelo",
   "catalogue_cataloguesListHeading": "Kalustoluettelot",
-  "catalogue_showGrouped": "Näytä ryhmissä",
+  "catalogue_showGrouped": "Näytä ryhmäytettyinä (Kylkinumero - Malli - Tyyppi)",
```

\+ rekisteripäivämäärä

```suggestion
  "catalogue_showGrouped": "Näytä ryhmäytettyinä (Kylkinumero - Malli - Tyyppi - Rekisteripäivä)",
```